src/jvmMain/kotlin/io/beatmaps/api/playlist/search.kt

```diff
@@ -19,13 +19,7 @@ import io.beatmaps.common.api.EPlaylistType
 import io.beatmaps.common.db.PgConcat
 import io.beatmaps.common.db.greaterEqF
 import io.beatmaps.common.db.lessEqF
-import io.beatmaps.common.dbo.Playlist
-import io.beatmaps.common.dbo.User
-import io.beatmaps.common.dbo.curatorAlias
-import io.beatmaps.common.dbo.handleCurator
-import io.beatmaps.common.dbo.handleOwner
-import io.beatmaps.common.dbo.joinOwner
-import io.beatmaps.common.dbo.joinPlaylistCurator
+import io.beatmaps.common.dbo.*
```

Comment:
No stars pls

Reply:
huh, i don't remember adding that? must have somehow accidentally typed a star